multidim-interop/src/generator.ts

```diff
@@ -79,14 +86,27 @@ export async function buildTestSpecs(versions: Array<Version>): Promise<Array<Co
             muxer: test.muxer,
             security: test.sec,
         })
-    )).concat(quicQueryResults.concat(quicV1QueryResults).concat(webtransportQueryResults).map((test): ComposeSpecification => buildSpec(containerImages, {
-        name: `${test.id1} x ${test.id2} (${test.transport})`,
-        dialerID: test.id1,
-        listenerID: test.id2,
-        transport: test.transport,
-        muxer: "quic",
-        security: "quic",
-    })))
+    )).concat(
+        quicQueryResults
+        .concat(quicV1QueryResults)
+        .concat(webtransportQueryResults)
+        .map((test): ComposeSpecification => buildSpec(containerImages, {
+            name: `${test.id1} x ${test.id2} (${test.transport})`,
+            dialerID: test.id1,
+            listenerID: test.id2,
+            transport: test.transport,
+            muxer: "quic",
+            security: "quic",
+        })))
+        .concat(webrtcQueryResults
+        .map((test): ComposeSpecification => buildSpec(containerImages, {
```

Comment:
Can we indent this line? The map applies to `webrtcQueryResults` not the `concat(...)` expression like above.

Reply:
yup makes sense Marco, updated :)